cgmanifests/generated/cgmanifest.json

```diff
@@ -242,7 +242,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "5a5f8a5935762397aa68429b5493084ff970f774",
+          "commitHash": "579ef250b69d57ebc4507d4a8fa31a9648e7c952",
```

Comment:
Now we are using cmake external and Azure storage to get external git repos. Is this still needed?